x11-packages/weston/0002-drop-deps-libinput.patch

```diff
@@ -48,70 +11,116 @@
  #include <libevdev/libevdev.h>
  #include <linux/input.h>
  #include <sys/time.h>
-@@ -1046,6 +1048,7 @@
+@@ -1070,7 +1072,7 @@
+ 		load_pipewire(wet.compositor, wet.config);
  	}
  }
- 
+-
 +#ifndef __ANDROID__
  static int
  save_touch_device_calibration(struct weston_compositor *compositor,
  			      struct weston_touch_device *device,
-@@ -1103,6 +1106,7 @@
+@@ -1128,7 +1130,7 @@
  
  	return ret;
  }
+-
 +#endif
- 
  static const struct {
  	const char *name;
-@@ -1184,12 +1188,14 @@
+ 	enum require_outputs mode;
+@@ -1208,14 +1210,14 @@
+ 		else
  			compositor->use_color_manager = true;
  	}
- 
+-
 +#ifndef __ANDROID__
  	/* weston.ini [libinput] */
  	s = weston_config_get_section(config, "libinput", NULL, NULL);
  	weston_config_section_get_bool(s, "touchscreen_calibrator", &cal, 0);
  	if (cal)
  		weston_compositor_enable_touch_calibrator(ec,
  						save_touch_device_calibration);
+-
 +#endif
- 
  	return 0;
  }
-@@ -1905,6 +1911,7 @@
+ 
+@@ -2127,6 +2129,7 @@
  	}
  }
  
 +#ifndef __ANDROID__
  static void
  configure_input_device_accel(struct weston_config_section *s,
  		struct libinput_device *device)
-@@ -2106,6 +2113,7 @@
+@@ -2328,7 +2331,7 @@
  
  	configure_input_device_scroll(s, device);
  }
+-
 +#endif
- 
  static int
  drm_backend_output_configure(struct weston_output *output,
-@@ -3019,6 +3027,7 @@
+ 			     struct weston_config_section *section)
+@@ -3377,6 +3380,7 @@
  load_drm_backend(struct weston_compositor *c, int *argc, char **argv,
  		 struct weston_config *wc, enum weston_renderer_type renderer)
  {
 +#ifndef __ANDROID__
  	struct weston_drm_backend_config config = {{ 0, }};
  	struct weston_config_section *section;
  	struct wet_compositor *wet = to_wet_compositor(c);
-@@ -3081,6 +3090,11 @@
+@@ -3440,7 +3444,10 @@
+ 	free(config.gbm_format);
  	free(config.seat_id);
  	free(config.specific_device);
- 
+-
 +#else
 +	weston_log("error: drm_backend is not avaiable on Android.\n");
 +	abort();
 +#endif
-+
  	return 0;
  }
  
+diff -uNr weston-14.0.0/frontend/meson.build weston-14.0.0.mod/frontend/meson.build
```

Comment:
This block can be moved to the beginning of thie file, which can reduce the diff between the two commits.